packages/nimble-components/src/rich-text/viewer/index.ts

```diff
@@ -19,8 +19,10 @@ export class RichTextViewer extends RichText {
      *
      * @public
      * Markdown string to render its corresponding rich text content in the component.
+     * @remarks
+     * HTML Attribute: markdown
      */
-    @observable
+    @attr
```

Comment:
Let's use the [attr `fromView` configuration](https://fast.design/docs/getting-started/fast-element#attribute-bindings) instead of the default which is `reflect`. These are likely large strings and there isn't benefit to serializing the property assignments to attributes to reflect to the DOM. It's just extra overhead in well-behaved apps.